src/parser/core/modules/items/PrePotion.js

```diff
@@ -137,24 +156,26 @@ class PrePotion extends Analyzer {
   }
 
   potionAdjuster(specID) {
-    this.alternatePotion = STR_SPECS.includes(specID) ? ITEMS.BATTLE_POTION_OF_STRENGTH.id : ITEMS.BATTLE_POTION_OF_AGILITY.id;
-    if (AGI_SPECS.includes(specID)) {
+    this.alternatePotion = STR_SPECS.includes(specID) ? ITEMS.BATTLE_POTION_OF_STRENGTH.id : AGI_SPECS.includes(specID) ? ITEMS.BATTLE_POTION_OF_AGILITY.id : ITEMS.BATTLE_POTION_OF_INTELLECT.id;
```

Comment:
eww nested ternaries

Reply:
Yeah, best option I could think of in the moment 😅